src/screens/Events/EventPage.js

```diff
@@ -1,6 +1,9 @@
 // Setup
 import React, { Component } from "react";
+import { connect } from "react-redux";
 import { StyleSheet, View } from "react-native";
+import { Translate, withLocalize, getLanguages, getActiveLanguage, setActiveLanguage
```

Comment:
We do not use `Translate`, `getActiveLanguage`, or `setActiveLanguage` in this component, so we can remove those imports.